### DISCUSSION TITLE
Hackathon 2022

### DISCUSSION THREAD

**Discussion by UlisesNh** [Launch-X-Latam/MisionFrontEnd#520] [Team Posts]
¡Qué tal!, 
busco equipo para participar en el Hackathon. Soy de México, tengo conocimientos en HTML, CSS3 y JavaScript.

**Comment by CAMILOITT** [comment]
mi nombre es Camilo estoy interesado en hacer el grupo
correo: : lx11002@innovaccion.mx

**Comment by UlisesNh** [comment]
Claro, Camilo.. eres bienvenido!

**Comment by JosafatJimenezB** [comment]
Un compañero y yo podemos entrar?

**Comment by CGux8** [comment]
hola, puedo completar el grupo ?
este es mi correo cgux08@gmail.com
👍

**Comment by UlisesNh** [comment]
> Un compañero y yo podemos entrar?

Por supuesto!, les dejo mi correo lx16635@innovaccion.mx

**Comment by JosafatJimenezB** [comment]
> Un compañero y yo podemos entrar?
> 
> Por supuesto!, les dejo mi correo lx16635@innovaccion.mx

ya te envie mensaje por teams xd

**Comment by josa6f** [comment]
Hola compañeros, algún equipo que tenga cupo para mi, o algunos compañeros que quieran hacer un equipo conmigo, tengo conocimiento en requerimientos, UX/UI, HTML, CSS, JavaScript, y algo de MySQL soy de la Ciudad de México, mi correo es lx02172@innovaccion.mx, saludos

**Comment by JazminVR05** [comment]
Buen día compañeros, ¿Aún hay espacio para 2 integrantes más? Conocimiento ej HTML, css, JS, ingeniería en TI

**Comment by JazminVR05** [comment]
Dejo mi correo lx01675@innovaccion.mx

**Comment by LaBalaElmo** [comment]
Buenas, si necesitan un integrante estoy disponible, mi correo: lx10061@innovaccion.mx.
Un saludo
